storage-image-labeling/functions/lib/config.js

```diff
@@ -16,6 +16,6 @@
  */
```

Comment:
is the `storage-image-labeling` extension code supposed to be here?

Reply:
@Salakar @russellwheatley  Has this been resolved now. I assume it appeared when syncing with the next branch?